src/qspdlog_model.hpp

```diff
@@ -32,4 +34,5 @@ class QSpdLogModel : public QAbstractListModel
 
 private:
     std::vector<entry_t> _items;
+    int _maxEntries;
```

Comment:
I'm thinking if this could be an `std::optional<std::size_t>` instead.

Reply:
Honestly I never tried using std::optional and it looks like it really is useful in this situation, as you can now clearly see if there is a maximum or not.